### PR TITLE
Switch category of extensions to AI

### PR DESCRIPTION
**Pull request by cescoffier**
Waiting for "code.quarkus.io" to have an AI category (Ping @ia3andy).

### CODE REVIEW

**Review by geoand — APPROVED** [review]

**Comment by ia3andy** [comment]
@cescoffier I think this is the place to add it: https://github.com/quarkusio/quarkus/blob/main/devtools/bom-descriptor-json/src/main/resources/catalog-overrides.json

File: core/runtime/src/main/resources/META-INF/quarkus-extension.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ metadata:
     - langchain4j
   guide: "https://docs.quarkiverse.io/quarkus-langchain4j/dev/index.html"
   categories:
-    - "miscellaneous"
+    - "AI"
```
Comment (maxandersen) [inline core/runtime/src/main/resources/META-INF/quarkus-extension.yaml:10]: should be lower case "ai", its an ID not a name/text.

**Review by maxandersen — CHANGES_REQUESTED** [review]
use "ai" rather than "AI" then all matches other category ids

**Comment by maxandersen** [comment]
related https://github.com/quarkusio/quarkus/pull/41425

**Comment by cescoffier** [comment]
Switch to "ai" instead of "AI"

**Comment by geoand** [comment]
Is there something blocking this?

**Comment by cescoffier** [comment]
Nothing blocking.

**Pull request merged**